pkg/generate/config/config.go

```diff
@@ -86,6 +94,11 @@ type ResourceConfig struct {
 	ListOperation *ListOperationConfig `json:"list_operation,omitempty"`
 }
 
+// resource fields that map to custom operation
+type CustomUpdateOperationFields struct {
+	Fields []string `json:"fields"`
```

Comment:
Please add a docstring comment explaining what should go in the `fields` `[]string`.

Reply:
Added docstring.